crashreports/crashreporter/CrashReportApp.cpp

```diff
@@ -0,0 +1,461 @@
+#include "CrashReportApp.h"
+
+#include <sstream>
+#include <memory>
+
+#include <wx/cmdline.h>
+#include <wx/chartype.h>
+#include <wx/artprov.h>
+#include <wx/filename.h>
+#include <wx/stdpaths.h>
+
+#include "google_breakpad/processor/basic_source_line_resolver.h"
+#include "google_breakpad/processor/minidump_processor.h"
+#include "google_breakpad/processor/process_state.h"
+#include "google_breakpad/processor/minidump.h"
+#include "processor/stackwalk_common.h"
+
+#include "warning.xpm"
+
+//Temporary solution until lib-strings is added
+#define XC(msg, ctx) (_(msg))
```

Comment:
I understand you need another definition of XC if you don't include lib-strings.

But the ctx argument must not be ignored.  Otherwise, the runtime lookup of the string will not find the translation.  The string and the context must be passed to the wxTranslation system.

I think the macro should expand to

`wxGetTranslation( msg, wxString{}, ctx )`

If you rebase this work onto more recent master, you can change to one of the updated languages (Dutch, Hindi, Ukrainian, or Corsican) and test that it works.